cfpb/college-costs
# Graph work and minor Django template cleanup

This hides national average graph data and explanatory boxes for settlement schools. It also cleans up some unneeded Django template code that was previously used to make data available to the JS.
## Removals
- Django-passed `window` objects for school and program data. The JS hasn't been using the passed `school` and `program` data objects since February (https://github.com/cfpb/college-costs/pull/118), and Django just stopped passing the `national` data object (https://github.com/cfpb/college-costs/pull/159).
## Changes
- Hides national averages on the graph for settlement schools.
- Hides explanatory boxes for settlement schools.
## Testing
- Pull down the branch. `gulp`, and [load a sample settlement school](http://localhost:8000/paying-for-college2/understanding-your-financial-aid-offer/offer/?iped=449898&pid=1509&oid=fa8283b5b7c939a058889f997949efa566c616c6&tuit=44565&ppl=0&subl=3500&gpl=0&perl=5500&book=500&gib=0&pelg=1500&ta=500&unsl=6000&insi=5.0&parl=7000&insl=3000&stag=2000&prvl=17000&prvi=4.55&wkst=1000&hous=3000&othg=100&tran=500&schg=2000&mta=0&othr=500) in your browser.
- Make sure that you've [followed the directions in the README to setup up functional testing](https://github.com/cfpb/college-costs/), then run `protractor test/functional/conf.js`. You should see something reasonably close to the following:
  
  > 63 specs, 7 failures
  > Finished in 180.51 seconds
  
  The seven failures are due to the change in how BLS expenses are given in the API; the JS hasn't been updated yet to pick them up, but will be next sprint.
- **CAVEAT:** As reported on https://github.com/cfpb/college-costs/issues/120, the JS refactor in February is causing no data at all to be passed for schools besides the ones set up with sample program data. 😞  This makes it hard to test to make sure that it's working properly for both positive and negative cases. @mistergone is aware of this problem.
## Review
- @mistergone @higs4281 @amymok 
## Screenshots

An tweaked graph for an example settlement school, showing no national data or explanatory box:

![screen shot 2016-05-23 at 12 51 31 am](https://cloud.githubusercontent.com/assets/1183435/15460591/91baa5d0-2080-11e6-9577-fecacc153daf.png)
## Checklist
